src/internal/dockertestenv/etcd.go

```diff
@@ -0,0 +1,20 @@
+package dockertestenv
+
+import (
+	"testing"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/require"
+	etcd "go.etcd.io/etcd/client/v3"
+	"go.etcd.io/etcd/client/v3/mock/mockserver"
+)
+
+func NewTestEtcd(t *testing.T) *etcd.Client {
```

Comment:
We have a package `testetcd`.  Which provides an in memory etcd.  In this package I would expect this function to bring up a docker image.

Reply:
If you need this exact function, I would put it in `testetcd` and explain why it's different from the other functions in there, and when you should use it.  Otherwise take the client out of the EtcdEnv that you get from `testetcd`